Version: 0.1.0-alpha

ax3l opened the pull request:
Prepare the first developer preview, `0.1.0-alpha`.

I will create a GPG signed tag after that and draw a release, including a DOI on [rodare](https://rodare.hzdr.de/).

Later on, we will do a `stable` and `release-newVersion` branches but for now we just bump versions happily inside the `dev`.

@C0nsultant do you have an [ORCiD](https://orcid.org/) that I can use to identify you?

Related to #87

ax3l committed:
Version: 0.1.0-alpha1

Prepare the first developer preview.

anokfireball commented:
> do you have an ORCiD that I can use to identify you?

I have now: https://orcid.org/0000-0001-8704-1769

ax3l committed:
Docs: Table Of Supported Versions

ax3l committed:
Version: Unique without Suffix

0.1.0-alpha
0.1.1-alpha
0.1.2-alpha

...
0.X.0-beta
0.X.1-beta
...

1.0.0 (stable)

ax3l commented:
Made the version unique when suffix is dropped:
```
# non-semantic initial pre-releases
0.1.0-alpha
0.1.1-alpha
0.1.2-alpha

(there will not be a 0.1.Y release without -alpha suffix)

...
0.X.0-beta
0.X.1-beta

(there will not be a 0.X.Y release without -beta suffix)
...

# semantic releases, following `standardMAJOR.apiMAJOR.apiMINOR`
1.0.0 (stable)
```

ax3l commented:
just as a note: there is an issue with rodare and webhooks from GitHub. Will add the DOI to the release another time.
